lisa/analysis/cpus.py

```diff
@@ -86,6 +176,32 @@ def plot_context_switches(self):
             invert_axes=True,
         )
 
+    @TraceAnalysisBase.plot_method
+    @df_utils.used_events
+    def plot_states(self):
+        """
+        Plot stacked histogram of BUSY/IDLE states of each CPU.
```

Comment:
Instead of "BUSY/IDLE states", refer the the actual dataframe method:
```
Plot stacked histogram of per-CPU states returned by :meth:`df_states`
```